addons/movments_accounts_balances/controllers/controllers.py

```diff
@@ -199,20 +214,148 @@ def delete_bill(self, bill_id):
         except Exception as e:
             return "Failed to delete bill: {}".format(e)
 
+    ##create/get/delete_bills payment
+    @api.model
+    def create_bill_payment(self, bills_ids, journal_id, payment_method_line_id):
+        Payment = self.env['account.payment']
+        Bill = self.env['account.move']
+        bills = Bill.browse(bills_ids)
+
+        total_amount = sum(bill.amount_residual for bill in bills)
+        payment_vals = {
+            'amount': total_amount,
+            'partner_id': bills[0].partner_id.id,  # Assuming all bills are for the same partner
+            'partner_type': 'supplier',
+            'payment_type': 'outbound',
+            'journal_id': journal_id,
+            'payment_method_line_id': payment_method_line_id,
+
+        }
+
+        payment = Payment.create(payment_vals)
+        payment.action_post()
+
+        # Reconcile each bill with the payment
+        # for bill in bills:
+        #     payment.register_payment(bill.invoice_payments_widget)
+
+        return payment.id
+
+    @api.model
+    def get_bill_payment(self, payment_id):
```

Comment:
Remove this method. We only need to support `get_bill_payment_by_journal_entry_id` ?